src/app/utilities/currency-util.ts

```diff
@@ -33,3 +33,5 @@ export function centsToBaseCurrencyUnit(cents: number) {
 export function baseCurrencyUnitToCents(base: number) {
     return Math.floor(base * 100);
 }
+
+
```

Comment:
Blank lines. :) 

In addition, always leave 1 blank line after your last line. There are [some reasons](https://thoughtbot.com/blog/no-newline-at-end-of-file#:~:text=So%2C%20it%20turns%20out%20that,is%20a%20line%20%E2%80%9Cterminator%E2%80%9D.) defending that statement.